include/model.php

```diff
@@ -122,6 +122,16 @@ public function has_languages() {
 	 * @return array List of PLL_Language objects or PLL_Language object properties.
 	 */
 	public function get_languages_list( $args = array() ) {
+		if ( ! $this->are_languages_ready() ) {
+			/* translators: %s is the name of a hook. */
+			$message = function_exists( '__' ) ? __( 'This function must not be called before the hook %s.', 'polylang' ) : 'This function must not be called before the hook %s.';
```

Comment:
I think we chose to not translate error message, am I wrong @Chouby?

Reply:
Exactly. Unlike WP, we don't translate messages displayed to developers only.

Reply:
Done in [7ddbfa8](https://github.com/polylang/polylang/pull/1211/commits/7ddbfa870ea94a405822d3f75a637c20cd5150fb).